packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_stats_table.dart

```diff
@@ -23,27 +24,28 @@ class _ClassNameColumn extends ColumnData<HeapStatsRecord> {
         );
 
   @override
-  String? getValue(HeapStatsRecord record) => record.heapClass.className;
+  String? getValue(HeapClassStatistics record) => record.heapClass.className;
```

Comment:
r/record/classStats - here and elsewhere now that the class name has changed from HeapStatsRecord to HeapClassStatistics